File: BehavioralDesignPattern/StrategyPattern/java/Driver.java
package BehavioralDesignPattern.StrategyPattern.java;

import BehavioralDesignPattern.StrategyPattern.java.ConcreteStrategy.CreditCardStrategy;
import BehavioralDesignPattern.StrategyPattern.java.ConcreteStrategy.DebitCardStrategy;
import BehavioralDesignPattern.StrategyPattern.java.ConcreteStrategy.PaypalStrategy;

public class Driver {
    public static void main(String[] args){
        Context paymentContext = new Context(new CreditCardStrategy());
        paymentContext.pay(100);

        paymentContext = new Context(new PaypalStrategy());
        paymentContext.pay(200);

        paymentContext = new Context(new DebitCardStrategy());
        paymentContext.pay(300);
    }
}

File: BehavioralDesignPattern/StrategyPattern/python/Driver.py
from Context import Context
from ConcreteStrategy.CreditCardStrategy import CreditCardStrategy
from ConcreteStrategy.DebitCardStrategy import DebitCardStrategy
from ConcreteStrategy.PaypalStrategy import PaypalStrategy

strategy = CreditCardStrategy()
context = Context(strategy)
context.pay(100)

strategy = DebitCardStrategy()
context = Context(strategy)
context.pay(200)

strategy = PaypalStrategy()
context = Context(strategy)
context.pay(100)